<!DOCTYPE html>
<html>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<head>
  <link rel="stylesheet" type="text/css" href="stylesheet.css">
</head>
<body class="science">
<div class="main">
<h1>Brandon Seah</h1>
<h2>Eukaryotic Microbiology • Microbial Symbiosis</h2>

<p><a href="http://orcid.org/0000-0002-1878-4363">ORCiD</a> |
 <a href="https://scholar.google.de/citations?user=3l8G5BwAAAAJ&hl=en">Google Scholar</a> |
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=seah+bk%5Bau%5D">PubMed</a> |
 <a href="http://github.com/kbseah">GitHub</a></p>

<p>I am a postdoctoral researcher on ciliate genomics in the group of
<a href="https://scholar.google.de/citations?user=qUOL-gEAAAAJ&hl=en&oi=ao">
Estienne Swart</a> at the <a href="http://www.eb.tuebingen.mpg.de">Max Planck
Institute for Developmental Biology</a> in Tübingen, Germany.</p>

<p>My doctoral work was performed at the <a href="https://www.mpi-bremen.de/">
Max Planck Institute for Marine Microbiology</a> in Bremen, Germany, where I
worked closely with <a href="http://hgrubervodicka.weebly.com">Harald Gruber-
Vodicka</a> in the <a href="https://www.mpi-bremen.de/en/Department-of-Symbiosis.html">
Symbiosis Department</a> headed by Prof Nicole Dubilier.</p>

<img src="icons/magglass.png" class="marginal"><h1>Research Interests</h1>

<p>My research interests lie in protists (microbial eukaryotes), microbial
symbiosis, and in applying bioinformatics to tackle problems in genomics,
phylogenetics, and organismal biology.</p>

<h2><i>Kentrophoros</i> and its bacterial symbionts</h2>

<p>The unusual ciliate <i>Kentrophoros</i> and its ectosymbiotic bacteria were
the focus of my doctoral research. <i>Kentrophoros</i> are found in marine
sediment (muds, fine sand) of coastal areas around the world. The bacterial
symbionts are attached to the ciliate’s surface, and use chemical energy from
the environment (e.g. from oxidation of sulfide) to build up biomass, a process
known as chemosynthesis. The ciliates then harvest them for food, and so the
symbionts have been called a “microbial kitchen garden”.</p>

<p>My research showed that the symbionts are a distinct lineage of
Gammaproteobacteria, which we called “<i>Candidatus</i> Kentron”, that are
uniquely associated with these <i>Kentrophoros</i> ciliates (<a
href="https://doi.org/10.1098/rspb.2017.0764">Seah et al. 2017</a>). However,
unlike all other chemosynthetic symbioses known to date, these symbionts do not
encode metabolic pathways for autotrophic CO<sub>2</sub> fixation (<a
href="https://doi.org/10.1128/mBio.01112-19">Seah et al. 2019</a>). Instead of
being a “garden” for their hosts, they are assimilating organic molecules from
the environment like acetate and propionate, “recycling” these low-value waste
products of fermentation into high-value biomass that their hosts then
consume.</p>

<p>This makes it an exception to the usual textbook explanations of such
symbioses as autotrophic CO<sub>2</sub>-fixing factories. Instead, alternative
carbon sources and metabolic flexibility in storing and mobilizing resources
are important for <i>Kentrophoros</i>, and other symbiotic systems too (<a
href="https://doi.org/10.1073/pnas.1818995116">Jäckle et al. 2019</a>).</p>

<br />

<span class="centered">
<iframe src="https://player.vimeo.com/video/89605962" width="500" height="281"
  frameborder="0" webkitallowfullscreen mozallowfullscreen
  allowfullscreen></iframe>
<p><a href="https://vimeo.com/89605962"><i>Kentrophoros</i> -- The Movie</a>
from <a href="https://vimeo.com/kbseah">brandon seah</a> on <a
href="https://vimeo.com">Vimeo</a>.</p>
</span>

<h2>Bioinformatics tools</h2>

<p>Several software tools that I have been involved in developing have been
packaged for use by the wider community. These include:
<ul>
  <li><a href="https://github.com/kbseah/mass2adduct">mass2adduct</a>, an R package for analyzing matrix adducts in mass spectrometry imaging data</li>
  <li><a href="http://hrgv.github.io/phyloFlash/">phyloFlash</a>, a Perl pipeline to rapidly screen Illumina data for their SSU rRNA-based taxonomic composition (<a href="https://doi.org/10.1101/521922">Gruber-Vodicka, Seah, &amp; Pruesse 2020</a>)</li>
  <li><a href="https://github.com/kbseah/genome-bin-tools">gbtools</a>, an R package for exploring metagenomic assemblies (<a href="https://doi.org/10.3389/fmicb.2015.01451">Seah &amp; Gruber-Vodicka 2015</a>)</li>
</ul>

<img src="icons/document_icon.png" class="marginal"><h1>Publications</h1>
<p>Links to associated data or software are provided for publications where I
was the first or corresponding author; first-author publications in bold.</p>
<p>
<img src="icons/oa-logo-plos.png" alt="Open access logo" height=16px> open-access |
 <img src="icons/pencil_icon.png" alt="pencil" height=16px> preprint |
 * joint authorship |
 <sup>+</sup> corresponding
</p>

<ul class="bibliography">
<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1128/mSystems.00920-20">phyloFlash: Rapid SSU rRNA profiling and targeted assembly from metagenomes.</a></span><br />
  Gruber-Vodicka HR *, Seah BKB *, Pruesse E. <br />
  <details><summary><i>mSystems</i> 5, e00920-20. (2020)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1128/mSystems.00920-20">10.1128/mSystems.00920-20</a></span> |
  <span class="biblinks">pmid: <a href="https://pubmed.ncbi.nlm.nih.gov/33109753/">33109753</a></span> |
  <span class="biblinks">pmc: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7593591/">PMC7593591</a></span> |
  <span class="biblinks"><a href="https://doi.org/10.1101/521922">preprint</a></span>
  <ul class="datalist">
    <li>Software <a href="https://github.com/HRGV/phyloFlash">phyloFlash</a>. Zenodo doi: <a href="https://doi.org/10.5281/zenodo.1327399">10.5281/zenodo.1327399</a></li>
    <li>Data supplements, Zenodo: <a href="https://doi.org/10.5281/zenodo.1464894">10.5281/zenodo.1464894</a>, <a href="https://doi.org/10.5281/zenodo.3909395">10.5281/zenodo.3909395</a>, <a href="https://doi.org/10.5281/zenodo.3909401">10.5281/zenodo.3909401</a>, <a href="https://doi.org/10.5281/zenodo.3909387">10.5281/zenodo.3909387</a>, <a href="https://doi.org/10.5281/zenodo.3909378">10.5281/zenodo.3909378</a>, <a href="https://doi.org/10.5281/zenodo.3909384">10.5281/zenodo.3909384</a> </li>
  </ul>
  </details>
</li>
<li data-ref-access="preprint">
  <span class="bibtitle"><a href="https://doi.org/10.1101/2020.07.01.183566"><i>Desulfovibrio diazotrophica</i> sp. nov., a sulphate reducing bacterium from the human gut capable of nitrogen fixation.</a></span><br />
  Sayavedra L, Li TQ, Batista MB, Seah BKB, Booth C, Zhai QX, Chen W, Narbad A. <br />
  <details><summary>bioRxiv. (2020, preprint)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1101/2020.07.01.183566">10.1101/2020.07.01.183566</a></span>
  </details>
</li>
<li data-ref-access="preprint">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1101/2020.03.19.998534"><i>Kentrophoros magnus</i> sp. nov. (Ciliophora, Karyorelictea), a new flagship species of marine interstitial ciliates.</a></span><br />
  Seah BKB<sup>+</sup>, Volland JM, Leisch N, Schwaha T, Dubilier N, Gruber-Vodicka HR. <br />
  <details><summary>bioRxiv. (2020, preprint)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1101/2020.03.19.998534">10.1101/2020.03.19.998534</a></span>
  </details>
</li>
<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1128/mBio.01112-19">Sulfur-oxidizing symbionts without canonical genes for autotrophic CO<sub>2</sub> fixation.</a></span><br />
  Seah BKB<sup>+</sup>, Antony CP, Huettel B, Zarzycki J, Schada von Borzyskowski L, Erb TJ, Kouris A, Kleiner M, Liebeke M, Dubilier N, Gruber-Vodicka HR. <br />
  <details><summary><i>mBio</i> 10, e01112-19 (2019)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1128/mBio.01112-19">10.1128/mBio.01112-19</a></span> |
  <span class="biblinks">pmid: <a href="https://pubmed.ncbi.nlm.nih.gov/31239380/">3123938</a></span> |
  <span class="biblinks">pmc: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6593406/">PMC6593406</a></span> |
  <span class="biblinks"><a href="https://doi.org/10.1101/540435">preprint</a></span>
  <ul class="datalist">
    <li>Supplementary information. Zenodo doi: <a href="https://doi.org/10.5281/zenodo.2555833">10.5281/zenodo.2555833</a></li>
    <li>Metagenome data. ENA, <a href="https://www.ebi.ac.uk/ena/data/view/PRJEB25374">PRJEB25374</a></li>
    <li>Metatranscriptome data. ENA, <a href="https://www.ebi.ac.uk/ena/data/view/PRJEB25540">PRJEB25540</a></li>
    <li>Metaproteome data. PRIDE, <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD011616">PXD011616</a></li>
    <li>Genome annotations. JGI GOLD <a href="https://gold.jgi.doe.gov/study?id=Gs0114545">Gs0114545</a></li>
  </ul>
  </details>
</li>
<li data-ref-access="oa">
  <span class="bibtitle"><a href="https://doi.org/10.1073/pnas.1818995116">Chemosynthetic symbiont with a drastically reduced genome serves as primary energy storage in the marine flatworm <i>Paracatenula</i>.</a></span><br />
  Jäckle O, Seah BKB, Tietjen M, Leisch N, Liebeke M, Kleiner M, Berg JS, Gruber-Vodicka HR.<br />
  <details><summary><i>Proceedings of the National Academy of Sciences, USA</i>, 116 (17), 8505-8514 (2019)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1073/pnas.1818995116">10.1073/pnas.1818995116</a></span> |
  <span class="biblinks">pmid: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30962361">30962361</a></span> |
  <span class="biblinks">pmc: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6486704/">PMC6486704</a> </span>
  </details>
</li>
<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1098/rspb.2017.0764">Specificity in diversity: single origin of a widespread ciliate-bacteria symbiosis.</a></span><br />
  Seah BKB<sup>+</sup>, Schwaha T, Volland JM, Huettel B, Dubilier N, Gruber-Vodicka H. <br />
  <details><summary><i>Proceedings of the Royal Society B</i> 284 (1858), 20170764. (2017)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1098/rspb.2017.0764">10.1098/rspb.2017.0764</a></span> |
  <span class="biblinks">pmid: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28701560">28701560</a></span> |
  <span class="biblinks">pmc: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5524500/">PMC5524500</a></span>
  <ul class="datalist">
    <li>3D imaging. Dryad, doi: <a href="https://doi.org/10.5061/dryad.nc5dp">10.5061/dryad.nc5dp</a></li>
    <li>Phylogenetic trees. TreeBASE, accession <a href="https://treebase.org/treebase-web/search/study/summary.html?id=19762">S19762</a></li>
    <li>SSU rRNA sequences. ENA, <a href="https://www.ebi.ac.uk/ena/data/view/LT621756-LT621967">LT621756 to LT621967</a> and <a href="https://www.ebi.ac.uk/ena/data/view/LT621968-LT622020">LT621968 to LT622020</a></li>
  </ul>
  </details>
</li>
<li data-ref-access="oa">
  <span class="bibtitle"><a href="https://doi.org/10.1038/nmicrobiol.2016.195">Chemosynthetic sulphur-oxidizing symbionts of marine invertebrate animals are capable of nitrogen fixation.</a></span><br />
  Petersen JM, Kemper A, Gruber-Vodicka H, Cardini U, van der Geest M, Kleiner M, Bulgheresi S, Mußmann M, Herbold C, Seah BKB, Antony CP, Liu D, Belitz A, Weber M. <br />
  <details><summary><i>Nature Microbiology</i> 2, 16195. (2016)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1038/nmicrobiol.2016.195">10.1038/nmicrobiol.2016.195</a></span> |
  <span class="biblinks"><a href="http://www.nature.com/articles/s41564-018-0196-5">Author correction</a></span>
  </details>
</li>
<li>
  <span class="bibtitle"><a href="https://doi.org/10.1016/j.syapm.2016.08.004">High rates of microbial dinitrogen fixation and sulfate reduction associated with the Mediterranean seagrass <i>Posidonia oceanica</i>.</a></span><br />
  Lehnen N, Marchant H, Schwedt A, Milucka J, Lott C, Weber M, Dekaezemacker J, Seah BKB, Hach PF, Mohr W, Kuypers MMM. <br />
  <details><summary><i>Systematic and Applied Microbiology</i> 39 (7), 476-483. (2016)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1016/j.syapm.2016.08.004">10.1016/j.syapm.2016.08.004</a></span>
  </details>
</li>
<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.3389/fmicb.2015.01451">gbtools: Interactive visualization of metagenome bins in R.</a></span><br />
  Seah BKB, Gruber-Vodicka HR. <br />
  <details><summary><i>Frontiers in Microbiology</i> 6, 01451. (2015)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.3389/fmicb.2015.01451">10.3389/fmicb.2015.01451</a></span> |
  <span class="biblinks">pmid: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26732662">26732662</a></span> |
  <span class="biblinks">pmc: <a href="https://europepmc.org/articles/PMC4683177">PMC4683177</a></span>
  <ul class="datalist">
    <li>Software <a href="https://github.com/kbseah/genome-bin-tools/">genome-bin-tools</a>. Zenodo, doi: <a href="https://doi.org/10.5281/zenodo.593084">10.5281/zenodo.593084</a>
  </ul>
  </details>
</li>
<li>
  <span class="bibtitle"><a href="https://doi.org/10.18968/jhbl.96.0_223">Two new moss species, <i>Trichosteleum fleischeri</i> and <i>Splachnobryum temasekensis</i>, from Singapore.</a></span></br />
  Tan BC, Ho BC, Seah BKB.<br />
  <details><summary><i>Journal of the Hattori Botanical Laboratory</i> 96, 223-230. (2004)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.18968/jhbl.96.0_223">10.18968/jhbl.96.0_223</a></span>
  </details>
</li>
</ul>

<img src="icons/chalkboard.png" class="marginal"><h1>Teaching and Outreach</h1>

<ul class="indentlist">
  <li><a href="https://github.com/kbseah/perlcourse2018">Course materials</a> for a three-day introductory workshop on Perl programming, originally prepared with my colleagues Lizbeth Sayavedra and Juliane Wippler.</li>
  <li><a href="https://kbseah.github.io/gbtlite/">gbtlite</a> – Browser-based visualization for metagenomics, developed as a teaching tool to accompany lectures on metagenomics delivered by my colleague Harald Gruber-Vodicka in the MarMic MSc course in marine microbiology.</li>
  <li><a href="https://sgprotist.wordpress.com">Protists in Singapore – Illustrated guide to microscopic life in the city</a> – The first web guide to eukaryotic microbes in Singapore, produced in 2011 with support from friends at the <a href="http://www.dbs.nus.edu.sg/lab/FW_lab/Main.htm">Freshwater and Invasion Biology Lab</a> at the National University of Singapore. </li>
  <li><a href="https://vimeo.com/kbseah">Vimeo account</a> – Where I post occasional videos of microorganisms that I encounter during my research.</li>
  <li>Illustrations of symbiotic marine fauna, free for reuse under Creative Commons CC BY-SA 4.0 license: 
  <a href="https://commons.wikimedia.org/wiki/File:Astomonema_icon.svg"><i>Astomonema</i></a>,
  <a href="https://commons.wikimedia.org/wiki/File:Bathymodiolus_icon.svg"><i>Bathymodiolus</i></a>,
  <a href="https://commons.wikimedia.org/wiki/File:Kentrophoros_icon.svg"><i>Kentrophoros</i> (1)</a>,
  <a href="https://commons.wikimedia.org/wiki/File:Kentrophoros_icon_2.svg"><i>Kentrophoros</i> (2)</a>,
  <a href="https://commons.wikimedia.org/wiki/File:Lucinid_icon.svg">Lucinidae</a>,
  <a href="https://commons.wikimedia.org/wiki/File:Olavius_icon.svg"><i>Olavius</i></a>,
  <a href="https://commons.wikimedia.org/wiki/File:Paracatenula_icon.svg"><i>Paracatenula</i></a>,
  <a href="https://commons.wikimedia.org/wiki/File:Riftia_pachyptila_icon.svg"><i>Riftia pachyptila</i></a>
  <a href="https://commons.wikimedia.org/wiki/File:Seagrass_icon.svg">Seagrass</a>,
  <a href="https://commons.wikimedia.org/wiki/File:Stilbonematine_icon.svg">Stilbonematinae</a>,
  <a href="https://commons.wikimedia.org/wiki/File:Trichoplax_icon.svg"><i>Trichoplax</i></a>
  </li>
</ul>

</div>

<div class="endmatter">
  <hr />
  <p><a href="index.html">Back</a></p>
</div>

</body>
</html>

<!--
Open access logo: https://commons.wikimedia.org/wiki/File:Open_Access_logo_PLoS_white.svg
By art designer at PLoS, modified by Wikipedia users Nina, Beao, and JakobVoss (http://www.plos.org/) [CC0], via Wikimedia Commons
-->
