<!DOCTYPE html>
<html>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<head>
  <link rel="stylesheet" type="text/css" href="stylesheet.css">
</head>
<body class="science">
<div class="main">
<h1>Brandon Seah</h1>
<h2>Eukaryotic Microbiology • Microbial Symbiosis</h2>

<p><a href="http://orcid.org/0000-0002-1878-4363">ORCiD</a> |
 <a href="https://scholar.google.de/citations?user=3l8G5BwAAAAJ&hl=en">Google Scholar</a> |
 <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=seah+bk%5Bau%5D">PubMed</a> |
 <a href="http://github.com/kbseah">GitHub</a></p>

<p>I am a research scientist at the <a
    href="https://www.thuenen.de/de/fachinstitute/biodiversitaet">Thünen
    Institute for Biodiversity</a> in Braunschweig, Germany.</p>

<p>Previously, I worked on ciliate genomics as a postdoc in the group of <a
  href="https://scholar.google.de/citations?user=qUOL-gEAAAAJ&hl=en&oi=ao">
  Estienne Swart</a> at the <a href="http://www.eb.tuebingen.mpg.de">Max Planck
  Institute for Biology</a> in Tübingen, Germany, and on microbial symbiosis
as a doctoral student at the <a href="https://www.mpi-bremen.de/"> Max Planck
  Institute for Marine Microbiology</a> in Bremen, Germany, in the team of <a
  href="http://hgrubervodicka.weebly.com">Harald Gruber-Vodicka</a>, under the
<a href="https://www.mpi-bremen.de/en/Department-of-Symbiosis.html"> Symbiosis
  Department</a> headed by Prof Nicole Dubilier.</p>

<img src="icons/magglass.png" class="marginal"><h1>Projects</h1>

<p>My research interests lie in protists (microbial eukaryotes), microbial
symbiosis, and in applying bioinformatics to tackle problems in genomics,
phylogenetics, and organismal biology.</p>

<h2>Ciliate Genomics</h2>

<p>Ciliates are microbial eukaryotes (protists). As their name suggests, they
are covered in cilia, but what actually makes them unique is the intricate
genomic ballet they perform during their sexual life cycle. Each cell has two
types of nuclei: germline micronuclei (MIC) which are like inactive “backup
copies” of the genome, and somatic macronuclei (MAC), “working copies” where
gene expression typically occurs.  MICs participate in meiosis and sexual
recombination, and some of the resulting daughter nuclei differentiate into
MACs. This process involves eliminating thousands of interspersed segments from
the genome called internally eliminated sequences (IESs), as well as chromosome
fragmentation, rearrangement, and copy number amplification. The genome content
of a MAC is hence a subset of what is found in the MIC.</p>

<p>This picture of the ciliate genome is based on a handful of model species.
To better understand how this complex developmental pathway has evolved and
where IESs come from, my research looks at the germline genomes of less well
studied ciliates, which entails developing new methods to assemble and
reconstruct germline genomes (<a
href="https://doi.org/10.1093/bioinformatics/btab613">Seah and Swart,
2021</a>). In the heterotrich <i>Blepharisma</i> (<a
href="https://doi.org/10.1073/pnas.2213985120">Seah et al., 2023</a>), we
found that miniature inverted-repeat transposable elements (MITEs), a type of
small mobile element, make up a substantial fraction of the IESs, including one
family that appears to have proliferated recently.</p>

<p>Another research theme is ambiguous genetic codes with context-dependent
stop/sense codons, where a stop codon can either terminate translation or
encode an amino acid. We showed that they are not isolated or rare phenomena,
but that such a code is used by a group of ciliates comprising over a hundred
described species, the karyorelicts
(<a href="https://doi.org/10.24072/pcjournal.141">Seah, Singh, and Swart, 2022</a>).</p>


<h2><i>Kentrophoros</i> and its Bacterial Symbionts</h2>

<br />

<span class="centered">
<iframe src="https://player.vimeo.com/video/89605962" width="500" height="281"
  frameborder="0" webkitallowfullscreen mozallowfullscreen
  allowfullscreen></iframe>
<p><a href="https://vimeo.com/89605962"><i>Kentrophoros</i> -- The Movie</a>
from <a href="https://vimeo.com/kbseah">brandon seah</a> on <a
href="https://vimeo.com">Vimeo</a>.</p>
</span>

<p>The unusual ciliate <i>Kentrophoros</i> and its ectosymbiotic bacteria were
the focus of my doctoral research. <i>Kentrophoros</i> are found in marine
sediment (muds, fine sand) of coastal areas around the world. The bacterial
symbionts are attached to the ciliate’s surface, and use chemical energy from
the environment (e.g. from oxidation of sulfide) to build up biomass, a process
known as chemosynthesis. The ciliates then harvest them for food, and so the
symbionts have been called a “microbial kitchen garden”.</p>

<p>My research showed that the symbionts are a distinct lineage of
Gammaproteobacteria, which we called “<i>Candidatus</i> Kentron”, that are
uniquely associated with these <i>Kentrophoros</i> ciliates (<a
href="https://doi.org/10.1098/rspb.2017.0764">Seah et al. 2017</a>). However,
unlike all other chemosynthetic symbioses known to date, these symbionts do not
encode metabolic pathways for autotrophic CO<sub>2</sub> fixation (<a
href="https://doi.org/10.1128/mBio.01112-19">Seah et al. 2019</a>). Instead of
being a “garden” for their hosts, they are assimilating organic molecules from
the environment like acetate and propionate, “recycling” these low-value waste
products of fermentation into high-value biomass that their hosts then
consume.</p>

<p>This makes it an exception to the usual textbook explanations of such
symbioses as autotrophic CO<sub>2</sub>-fixing factories. Instead, alternative
carbon sources and metabolic flexibility in storing and mobilizing resources
are important for <i>Kentrophoros</i>, and other symbiotic systems too (<a
href="https://doi.org/10.1073/pnas.1818995116">Jäckle et al. 2019</a>).</p>


<h2>Bioinformatics Tools</h2>

<p>Several software tools that I have been involved in developing have been
packaged for use by the wider community. These include:
<ul>
  <li><a href="https://github.com/Swart-lab/bleties">BleTIES</a>, a Python package for targeted reassembly of germline-limited sequences in ciliates from long-read sequencing data (<a href="https://doi.org/10.1093/bioinformatics/btab613">Seah &amp; Swart, 2021)</li>
  <li><a href="https://github.com/kbseah/mass2adduct">mass2adduct</a>, an R package for analyzing matrix adducts in mass spectrometry imaging data (<a href="https://doi.org/10.1021/acs.analchem.0c04720">Janda et al., 2021</a>)</li>
  <li><a href="http://hrgv.github.io/phyloFlash/">phyloFlash</a>, a Perl pipeline to rapidly screen Illumina data for their SSU rRNA-based taxonomic composition (<a href="https://doi.org/10.1101/521922">Gruber-Vodicka, Seah, &amp; Pruesse, 2020</a>)</li>
  <li><a href="https://github.com/kbseah/genome-bin-tools">gbtools</a>, an R package for exploring metagenomic assemblies (<a href="https://doi.org/10.3389/fmicb.2015.01451">Seah &amp; Gruber-Vodicka, 2015</a>)</li>
</ul>

<img src="icons/document_icon.png" class="marginal"><h1>Publications and Data</h1>
<p>Links to associated data or software are provided for publications where I
was the first or corresponding author; first-author publication titles in bold.
Preprints are open-access but not peer-reviewed. Click on triangles to show
details and links to supplements.</p>
<p>
<img src="icons/oa-logo-diamond.png" alt="Diamond open access logo" height=16px><img src="icons/oa-logo-plos.png" alt="Gold open access logo" height=16px><img src="icons/oa-logo-green.png" alt="Green open access logo" height=16px> open-access |
 <img src="icons/pencil_icon.png" alt="pencil" height=16px> preprint |
 * joint authorship |
 <sup>+</sup> corresponding
</p>

<ul class="bibliography">

<li data-ref-access="preprint">
  <span class="bibtitle"><a href="https://doi.org/10.1101/2023.08.09.552620">ISWI1 complex proteins facilitate developmental genome editing in <i>Paramecium</i></a></span><br />
    Singh A, Häußermann L, Emmerich C, Nischwitz E, Seah BKB, Butter F, Nowacki M, Swart EC.<br />
  <details><summary>bioRxiv. (2023, preprint)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1101/2023.08.09.552620">10.1101/2023.08.09.552620</a></span>
  </details>
</li>


<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1016/j.tig.2023.02.016">When cleaning facilitates cluttering – genome editing in ciliates</a></span><br />
  Seah BKB, Swart EC.<br />
  <details><summary><i>Trends in Genetics</i>, 39 (5) 344-346 (2023)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1016/j.tig.2023.02.016">10.1016/j.tig.2023.02.016</a></span> | 
  <span class="biblinks">pmid: <a href="https://pubmed.ncbi.nlm.nih.gov/36949004/">36949004</a></span>
  </details>
</li>

<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1073/pnas.2213985120">MITE infestation accommodated by genome editing in the germline genome of the ciliate <i>Blepharisma</i></a></span><br />
  Seah BKB, Singh M, Emmerich C, Singh A, Woehle C, Huettel B, Byerly A, Stover NA, Sugiura M, Harumoto T, Swart EC.<br />
  <details><summary><i>Proceedings of the National Academy of Sciences, USA</i>, 120 (4) e2213985120 (2023)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1073/pnas.2213985120">10.1073/pnas.2213985120</a></span> | 
  <span class="biblinks">pmid: <a href="https://pubmed.ncbi.nlm.nih.gov/36669106/">36669106</a></span> | 
  <span class="biblinks">pmc: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/pmc9942856/">PMC9942856</a></span> | 
  <span class="biblinks"><a href="https://doi.org/10.1101/2022.05.02.489906">preprint</a></span>
  <ul class="datalist">
    <li>MIC genome data, ENA, <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB46944">PRJEB46944</a></li>
    <li>Small RNA sequencing, ENA, <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB47200">PRJEB47200</a></li>
    <li>IES predictions, EDMOND, doi: <a href="https://dx.doi.org/10.17617/3.83">10.17617/3.83</a></li>
    <li>Repeat family annotations, EDMOND, doi: <a href="https://dx.doi.org/10.17617/3.82">10.17617/3.82</a></li>
    <li>Alignment of Tc1/Mariner transposase domains, EDMOND, doi: <a href="https://doi.org/10.17617/3.JLWBFM">10.17617/3.JLWBFM</a></li>
  </ul>
  </details>
</li>

<li data-ref-access="oa">
  <span class="bibtitle"><a href="https://doi.org/10.1073/pnas.2213887120">Origins of genome-editing excisases as illuminated by the somatic genome of the ciliate <i>Blepharisma</i></a></span><br />
  Singh M, Seah BKB, Emmerich C, Singh A, Woehle C, Huettel B, Byerly A, Stover NA, Sugiura M, Harumoto T, Swart EC.<br />
  <details><summary><i>Proceedings of the National Academy of Sciences, USA</i>, 120 (4), e2213887120 (2023)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1073/pnas.2213887120">10.1073/pnas.2213887120</a></span> | 
  <span class="biblinks">pmid: <a href="https://pubmed.ncbi.nlm.nih.gov/36669098/">36669098</a></span> | 
  <span class="biblinks">pmc: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/pmc9942806/">PMC9942806</a></span> | 
  <span class="biblinks"><a href="https://doi.org/10.1101/2021.12.14.471607">preprint</a></span>
  </details>
</li>

<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1016/j.protis.2022.125905">Improved methods for bulk cultivation and fixation of <i>Loxodes</i> ciliates for fluorescence microscopy</a></span><br />
  Seah BKB<sup>+</sup>, Emmerich C, Singh A, Swart EC.<br />
<details><summary><i>Protist</i>, 173 (5), 125905 (2022)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1016/j.protis.2022.125905">10.1016/j.protis.2022.125905</a></span> |
  <span class="biblinks">pmid: <a href="https://pubmed.ncbi.nlm.nih.gov/36027633/">36027633</a></span> |
  <span class="biblinks"><a href="https://doi.org/10.1101/2022.05.29.493905">preprint</a></span>
  <ul class="datalist">
    <li>Soil extract medium data, EDMOND, doi: <a href="https://doi.org/10.17617/3.4ZQNBK">10.17617/3.4ZQNBK</a></li>
  </ul>
  </details>
</li>

<li data-ref-access="oa-dmnd">
  <span class="bibtitlefirst"><a href="https://doi.org/10.24072/pcjournal.141">Karyorelict ciliates use an ambiguous genetic code with context-dependent stop/sense codons.</a></span><br />
  Seah BKB<sup>+</sup>, Singh A, Swart EC.<br />
  <details><summary><i>Peer Community Journal</i>, 2, e42 (2022)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.24072/pcjournal.141">10.24072/pcjournal.141</a></span> | 
  <span class="biblinks"><a href="https://doi.org/10.1101/2022.04.12.488043">preprint</a></span>
  <ul class="datalist">
    <li>Peer reviews, PCI Genomics, doi: <a href="https://doi.org/10.24072/pci.genomics.100019">10.24072/pci.genomics.100019</a></li>
    <li>Comparative dataset accessions, EDMOND, doi: <a href="https://doi.org/10.17617/3.XWMBKT">10.17617/3.XWMBKT</a></li>
    <li>18S rRNA phylogeny, EDMOND, doi: <a href="https://doi.org/10.17617/3.QLWR38">10.17617/3.QLWR38</a></li>
    <li>Transcriptome data, ENA, <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB50648">PRJEB50648</a></li>
    <li>Workflow <a href="https://github.com/Swart-lab/karyocode-workflow">karyocode-workflow</a>. Zenodo doi: <a href="https://doi.org/10.5281/zenodo.6647650">10.5281/zenodo.6647650</a> </li>
    <li>Workflow <a href="https://github.com/Swart-lab/karyocode-analysis-porc">karyocode-analysis-porc</a>. Zenodo doi: <a href="https://doi.org/10.5281/zenodo.6647652">10.5281/zenodo.6647652</a></li>
    <li>Workflow <a href="https://github.com/Swart-lab/karyocode-analysis-busco">karyocode-analysis-busco</a>. Zenodo doi: <a href="https://doi.org/10.5281/zenodo.6647679">10.5281/zenodo.6647679</a></li>
    <li>Software <a href="https://github.com/Swart-lab/PORC">PORC</a>. Zenodo doi: <a href="https://doi.org/10.5281/zenodo.6784075">10.5281/zenodo.6784075</a></li>
  </ul>
  </details>
</li>
<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1093/bioinformatics/btab613">BleTIES: Annotation of natural genome editing in ciliates using long read sequencing.</a></span><br />
  Seah BKB<sup>+</sup>, Swart EC. <br />
  <details><summary><i>Bioinformatics</i>, 37 (21), 3929-3931 (2021)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1093/bioinformatics/btab613">10.1093/bioinformatics/btab613</a></span> | 
  <span class="biblinks">pmid: <a href="https://pubmed.ncbi.nlm.nih.gov/34487139/">34487139</a></span> | 
  <span class="biblinks"><a href="https://doi.org/10.1101/2021.05.18.444610">preprint</a></span>
  <ul class="datalist">
    <li>Software <a href="https://github.com/Swart-lab/bleties">BleTIES</a>. Zenodo doi: <a href="https://doi.org/10.5281/zenodo.4723565">10.5281/zenodo.4723565</a></li>
    <li>Benchmarking tests (GitHub): <a href="https://github.com/Swart-lab/bleties-test-ptet">bleties-test-ptet</a>, <a href="bleties-test-tthe">bleties-test-tthe</a></li>
  </ul>
  </details>
</li>

<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1021/acs.analchem.0c04720">Determination of abundant metabolite matrix adducts illuminates the dark metabolome of MALDI-mass spectrometry imaging datasets.</a></span><br />
  Janda M*, Seah BKB*, Jakob D, Beckmann J, Geier B, Liebeke M. <br />
  <details><summary><i>Analytical Chemistry</i>, 93 (24), 8399–8407 (2021)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1021/acs.analchem.0c04720">10.1021/acs.analchem.0c04720</a></span> |
  <span class="biblinks">pmid: <a href="https://pubmed.ncbi.nlm.nih.gov/34097397/">34097397</a></span> |
  <span class="biblinks">pmc: <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/pmc8223199/">PMC8223199</a></span>
  <ul class="datalist">
    <li>Software <a href="https://github.com/kbseah/mass2adduct">mass2adduct</a>. Zenodo doi: <a href="https://doi.org/10.5281/zenodo.1405088">10.5281/zenodo.1405088</a></li>
    <li>Data supplement, Zenodo: <a href="https://doi.org/10.5281/zenodo.3363065">10.5281/zenodo.3363065</a></li>
    <li>Mass spectrometry imaging data: Metabolights <a href="https://www.ebi.ac.uk/metabolights/MTBLS954">MTBLS954</a></li>
  </ul>
  </details>
</li>

<li data-ref-access="oa">
  <span class="bibtitle"><a href="https://doi.org/10.1111/1462-2920.15538"><i>Desulfovibrio diazotrophica</i> sp. nov., a sulphate reducing bacterium from the human gut capable of nitrogen fixation.</a></span><br />
  Sayavedra L, Li TQ, Batista MB, Seah BKB, Booth C, Zhai QX, Chen W, Narbad A. <br />
  <details><summary><i>Environmental Microbiology</i>, 23 (6), 3164-3181 (2021)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1111/1462-2920.15538">10.1111/1462-2920.15538</a></span> | 
  <span class="biblinks"><a href="https://doi.org/10.1101/2020.07.01.183566">preprint</a></span> | 
  <span class="biblinks"><a href="https://doi.org/10.1111/1462-2920.16239">author correction</a></span>

  </details>
</li>

<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1128/mSystems.00920-20">phyloFlash: Rapid SSU rRNA profiling and targeted assembly from metagenomes.</a></span><br />
  Gruber-Vodicka HR *, Seah BKB *, Pruesse E. <br />
  <details><summary><i>mSystems</i> 5, e00920-20. (2020)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1128/mSystems.00920-20">10.1128/mSystems.00920-20</a></span> |
  <span class="biblinks">pmid: <a href="https://pubmed.ncbi.nlm.nih.gov/33109753/">33109753</a></span> |
  <span class="biblinks">pmc: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7593591/">PMC7593591</a></span> |
  <span class="biblinks"><a href="https://doi.org/10.1101/521922">preprint</a></span>
  <ul class="datalist">
    <li>Software <a href="https://github.com/HRGV/phyloFlash">phyloFlash</a>. Zenodo doi: <a href="https://doi.org/10.5281/zenodo.1327399">10.5281/zenodo.1327399</a></li>
    <li>Data supplements, Zenodo: <a href="https://doi.org/10.5281/zenodo.1464894">10.5281/zenodo.1464894</a>, <a href="https://doi.org/10.5281/zenodo.3909395">10.5281/zenodo.3909395</a>, <a href="https://doi.org/10.5281/zenodo.3909401">10.5281/zenodo.3909401</a>, <a href="https://doi.org/10.5281/zenodo.3909387">10.5281/zenodo.3909387</a>, <a href="https://doi.org/10.5281/zenodo.3909378">10.5281/zenodo.3909378</a>, <a href="https://doi.org/10.5281/zenodo.3909384">10.5281/zenodo.3909384</a> </li>
  </ul>
  </details>
</li>

<li data-ref-access="preprint">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1101/2020.03.19.998534"><i>Kentrophoros magnus</i> sp. nov. (Ciliophora, Karyorelictea), a new flagship species of marine interstitial ciliates.</a></span><br />
  Seah BKB<sup>+</sup>, Volland JM, Leisch N, Schwaha T, Dubilier N, Gruber-Vodicka HR. <br />
  <details><summary>bioRxiv. (2020, preprint)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1101/2020.03.19.998534">10.1101/2020.03.19.998534</a></span>
  </details>
</li>

<li data-ref-access="preprint">
  <span class="bibtitle"><a href="https://doi.org/10.5281/zenodo.3928556"><i>Kentrophoros</i> Field Manual</a></span><br />
  Seah BKB. <br />
  <details><summary>Technical report (not peer-reviewed). Zenodo. (2020)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.5281/zenodo.3928556">10.5281/zenodo.3928556</a></span>
  </details>
</li>

<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1128/mBio.01112-19">Sulfur-oxidizing symbionts without canonical genes for autotrophic CO<sub>2</sub> fixation.</a></span><br />
  Seah BKB<sup>+</sup>, Antony CP, Huettel B, Zarzycki J, Schada von Borzyskowski L, Erb TJ, Kouris A, Kleiner M, Liebeke M, Dubilier N, Gruber-Vodicka HR. <br />
  <details><summary><i>mBio</i> 10, e01112-19 (2019)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1128/mBio.01112-19">10.1128/mBio.01112-19</a></span> |
  <span class="biblinks">pmid: <a href="https://pubmed.ncbi.nlm.nih.gov/31239380/">3123938</a></span> |
  <span class="biblinks">pmc: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6593406/">PMC6593406</a></span> |
  <span class="biblinks"><a href="https://doi.org/10.1101/540435">preprint</a></span>
  <ul class="datalist">
    <li>Supplementary information. Zenodo doi: <a href="https://doi.org/10.5281/zenodo.2555833">10.5281/zenodo.2555833</a></li>
    <li>Metagenome data. ENA, <a href="https://www.ebi.ac.uk/ena/data/view/PRJEB25374">PRJEB25374</a></li>
    <li>Metatranscriptome data. ENA, <a href="https://www.ebi.ac.uk/ena/data/view/PRJEB25540">PRJEB25540</a></li>
    <li>Metaproteome data. PRIDE, <a href="https://www.ebi.ac.uk/pride/archive/projects/PXD011616">PXD011616</a></li>
    <li>Genome annotations. JGI GOLD <a href="https://gold.jgi.doe.gov/study?id=Gs0114545">Gs0114545</a></li>
  </ul>
  </details>
</li>

<li data-ref-access="oa">
  <span class="bibtitle"><a href="https://doi.org/10.1073/pnas.1818995116">Chemosynthetic symbiont with a drastically reduced genome serves as primary energy storage in the marine flatworm <i>Paracatenula</i>.</a></span><br />
  Jäckle O, Seah BKB, Tietjen M, Leisch N, Liebeke M, Kleiner M, Berg JS, Gruber-Vodicka HR.<br />
  <details><summary><i>Proceedings of the National Academy of Sciences, USA</i>, 116 (17), 8505-8514 (2019)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1073/pnas.1818995116">10.1073/pnas.1818995116</a></span> |
  <span class="biblinks">pmid: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30962361">30962361</a></span> |
  <span class="biblinks">pmc: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6486704/">PMC6486704</a> </span>
  </details>
</li>

<li data-ref-access="preprint">
  <span class="bibtitlefirst"><a href="http://nbn-resolving.de/urn:nbn:de:gbv:46-00106172-12">The bacterial ectosymbionts of the ciliate <i>Kentrophoros</i>.</a></span></br />
  Seah BKB<br />
  <details><summary>Doctoral dissertation, Universität Bremen (2017)</summary>
  <span class="biblinks">url: <a href="http://nbn-resolving.de/urn:nbn:de:gbv:46-00106172-12">http://nbn-resolving.de/urn:nbn:de:gbv:46-00106172-12</a></span>
  </details>
</li>

<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.1098/rspb.2017.0764">Specificity in diversity: single origin of a widespread ciliate-bacteria symbiosis.</a></span><br />
  Seah BKB<sup>+</sup>, Schwaha T, Volland JM, Huettel B, Dubilier N, Gruber-Vodicka H. <br />
  <details><summary><i>Proceedings of the Royal Society B</i> 284 (1858), 20170764. (2017)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1098/rspb.2017.0764">10.1098/rspb.2017.0764</a></span> |
  <span class="biblinks">pmid: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28701560">28701560</a></span> |
  <span class="biblinks">pmc: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5524500/">PMC5524500</a></span>
  <ul class="datalist">
    <li>3D imaging. Dryad, doi: <a href="https://doi.org/10.5061/dryad.nc5dp">10.5061/dryad.nc5dp</a></li>
    <li>Phylogenetic trees. TreeBASE, accession <a href="https://treebase.org/treebase-web/search/study/summary.html?id=19762">S19762</a></li>
    <li>SSU rRNA sequences. ENA, <a href="https://www.ebi.ac.uk/ena/data/view/LT621756-LT621967">LT621756 to LT621967</a> and <a href="https://www.ebi.ac.uk/ena/data/view/LT621968-LT622020">LT621968 to LT622020</a></li>
  </ul>
  </details>
</li>

<li data-ref-access="oa">
  <span class="bibtitle"><a href="https://doi.org/10.1038/nmicrobiol.2016.195">Chemosynthetic sulphur-oxidizing symbionts of marine invertebrate animals are capable of nitrogen fixation.</a></span><br />
  Petersen JM, Kemper A, Gruber-Vodicka H, Cardini U, van der Geest M, Kleiner M, Bulgheresi S, Mußmann M, Herbold C, Seah BKB, Antony CP, Liu D, Belitz A, Weber M. <br />
  <details><summary><i>Nature Microbiology</i> 2, 16195. (2016)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1038/nmicrobiol.2016.195">10.1038/nmicrobiol.2016.195</a></span> |
  <span class="biblinks"><a href="http://www.nature.com/articles/s41564-018-0196-5">author correction</a></span>
  </details>
</li>

<li>
  <span class="bibtitle"><a href="https://doi.org/10.1016/j.syapm.2016.08.004">High rates of microbial dinitrogen fixation and sulfate reduction associated with the Mediterranean seagrass <i>Posidonia oceanica</i>.</a></span><br />
  Lehnen N, Marchant H, Schwedt A, Milucka J, Lott C, Weber M, Dekaezemacker J, Seah BKB, Hach PF, Mohr W, Kuypers MMM. <br />
  <details><summary><i>Systematic and Applied Microbiology</i> 39 (7), 476-483. (2016)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.1016/j.syapm.2016.08.004">10.1016/j.syapm.2016.08.004</a></span>
  </details>
</li>

<li data-ref-access="oa">
  <span class="bibtitlefirst"><a href="https://doi.org/10.3389/fmicb.2015.01451">gbtools: Interactive visualization of metagenome bins in R.</a></span><br />
  Seah BKB, Gruber-Vodicka HR. <br />
  <details><summary><i>Frontiers in Microbiology</i> 6, 01451. (2015)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.3389/fmicb.2015.01451">10.3389/fmicb.2015.01451</a></span> |
  <span class="biblinks">pmid: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26732662">26732662</a></span> |
  <span class="biblinks">pmc: <a href="https://europepmc.org/articles/PMC4683177">PMC4683177</a></span>
  <ul class="datalist">
    <li>Software <a href="https://github.com/kbseah/genome-bin-tools/">genome-bin-tools</a>. Zenodo, doi: <a href="https://doi.org/10.5281/zenodo.593084">10.5281/zenodo.593084</a>
  </ul>
  </details>
</li>

<li data-ref-access="oa-dmnd">
  <span class="bibtitle"><a href="https://doi.org/10.18968/jhbl.96.0_223">Two new moss species, <i>Trichosteleum fleischeri</i> and <i>Splachnobryum temasekensis</i>, from Singapore.</a></span></br />
  Tan BC, Ho BC, Seah BKB.<br />
  <details><summary><i>Journal of the Hattori Botanical Laboratory</i> 96, 223-230. (2004)</summary>
  <span class="biblinks">doi: <a href="https://doi.org/10.18968/jhbl.96.0_223">10.18968/jhbl.96.0_223</a></span>
  </details>
</li>
</ul>

<img src="icons/chalkboard.png" class="marginal"><h1>Teaching and Outreach</h1>

<ul class="indentlist">
  <li><a href="https://github.com/kbseah/perlcourse2018">Course materials</a> for a three-day introductory workshop on Perl programming, originally prepared with my colleagues Lizbeth Sayavedra and Juliane Wippler.</li>
  <li><a href="https://kbseah.github.io/gbtlite/">gbtlite</a> – Browser-based visualization for metagenomics, developed as a teaching tool to accompany lectures on metagenomics delivered by my colleague Harald Gruber-Vodicka in the MarMic MSc course in marine microbiology.</li>
  <li><a href="https://sgprotist.wordpress.com">Protists in Singapore – Illustrated guide to microscopic life in the city</a> – The first web guide to eukaryotic microbes in Singapore, produced in 2011 with support from friends at the <a href="http://www.dbs.nus.edu.sg/lab/FW_lab/Main.htm">Freshwater and Invasion Biology Lab</a> at the National University of Singapore. </li>
  <li><a href="https://vimeo.com/kbseah">Vimeo account</a> – Where I post occasional videos of microorganisms that I encounter during my research.</li>
  <li>Illustrations of symbiotic marine fauna, free for reuse under Creative Commons CC BY-SA 4.0 license: 
  <a href="https://commons.wikimedia.org/wiki/File:Astomonema_icon.svg"><i>Astomonema</i></a>,
  <a href="https://commons.wikimedia.org/wiki/File:Bathymodiolus_icon.svg"><i>Bathymodiolus</i></a>,
  <a href="https://commons.wikimedia.org/wiki/File:Kentrophoros_icon.svg"><i>Kentrophoros</i> (1)</a>,
  <a href="https://commons.wikimedia.org/wiki/File:Kentrophoros_icon_2.svg"><i>Kentrophoros</i> (2)</a>,
  <a href="https://commons.wikimedia.org/wiki/File:Lucinid_icon.svg">Lucinidae</a>,
  <a href="https://commons.wikimedia.org/wiki/File:Olavius_icon.svg"><i>Olavius</i></a>,
  <a href="https://commons.wikimedia.org/wiki/File:Paracatenula_icon.svg"><i>Paracatenula</i></a>,
  <a href="https://commons.wikimedia.org/wiki/File:Riftia_pachyptila_icon.svg"><i>Riftia pachyptila</i></a>
  <a href="https://commons.wikimedia.org/wiki/File:Seagrass_icon.svg">Seagrass</a>,
  <a href="https://commons.wikimedia.org/wiki/File:Stilbonematine_icon.svg">Stilbonematinae</a>,
  <a href="https://commons.wikimedia.org/wiki/File:Trichoplax_icon.svg"><i>Trichoplax</i></a>
  </li>
</ul>

</div>

<div class="endmatter">
  <hr />
  <p><a href="index.html">Back</a></p>
</div>

</body>
</html>

<!--
Open access logo: https://commons.wikimedia.org/wiki/File:Open_Access_logo_PLoS_white.svg
By art designer at PLoS, modified by Wikipedia users Nina, Beao, and JakobVoss (http://www.plos.org/) [CC0], via Wikimedia Commons
-->
